packages/assets/package.json

```diff
@@ -2,12 +2,15 @@
   "name": "@react-native/assets-registry",
   "version": "0.73.0",
   "description": "Asset support code for React Native.",
+  "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git@github.com:facebook/react-native.git",
```

Comment:
`git@` (old) & `git://` (new) are more prone to attacks than `https://`.

>Due to the lack of TLS or other cryptography, cloning over `git://` _might lead to an arbitrary code execution vulnerability_, and should therefore be avoided unless you know what you are doing.

>
>If you run `git clone git://example.com/project.git`, an attacker who controls e.g your router can modify the repo you just cloned, inserting malicious code into it. If you then compile/run the code you just cloned, you will execute the malicious code. Running `git clone http://example.com/project.git` should be avoided for the same reason.

>
>Running `git clone https://example.com/project.git` _does not suffer from the same problem_ (unless the attacker can provide a TLS certificate for example.com). Running `git clone [git@example.com]:project.git` only suffers from this problem if you accept a wrong ssh key fingerprint.

Reference: https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols
It is very informative read 😃